tools/shared/src/main/scala/org/scalajs/core/tools/linker/frontend/BaseLinker.scala

```diff
@@ -115,7 +116,18 @@ final class BaseLinker(semantics: Semantics, esLevel: ESLevel, considerPositions
       }
 
       val linkingErrLevel = if (fatal) Level.Error else Level.Warn
-      analysis.errors.foreach(logError(_, logger, linkingErrLevel))
+      val maxDisplayErrors = {
+        val propName = "org.scalajs.core.tools.linker.maxlinkingerrors"
+        Try(System.getProperty(propName, "20").toInt).getOrElse(20).max(1)
```

Comment:
Our logger abstraction is much too generic to be bothered with such an option. How would you imagine configuring that at the logger level?